packages/frontend/app/templates/Adoption/PreAdoptionVisit/PreAdoptionVisit.jsx

```diff
@@ -0,0 +1,68 @@
+import Typography from "design-system/components/typography"
+import Feature from "design-system/patterns/feature"
+
+import Container from "design-system/components/container"
+import * as Styled from "./PreAdoptionVisit.styled"
+
+const values = [
+  {
+    heading: "Jak wygląda taka wizyta?",
+    description:
+      "Wizytator/ka sprawdza warunki dla zwierzęcia jakie panują w miejscu, gdzie zostanie zabrane zwierzę. ",
+    icon: "transparency",
+  },
+  {
+    heading: "Gdzie sie odbywa?",
+    description:
+      "Wizyta odbywa się w mieszkaniu lub domu potencjalnej rodziny, niezależnie od kraju zamieszkania rodziny.",
```

Comment:
```suggestion
      "Wizyta odbywa się w mieszkaniu lub domu potencjalnej rodziny, niezależnie od kraju zamieszkania.",
```
duplicated word "rodziny", I know it was designed this way 🙈